.github/workflows/update_ovs_image.yml

```diff
@@ -0,0 +1,21 @@
+name: Update the antrea/openvswitch image every 6 hours
+
+on:
+  schedule:
+    # every 6 hours
+    - cron: '0 */6 * * *'
```

Comment:
Any chance we can avoid repushing the image if no change? Computers use power ;)

Reply:
I changed the schedule interval from 6 to 12 hours.

I also did some researched and completely over-engineered the process... We now only rebuild the image if the base image has changed (or more precisely we use the already published image as a docker cache to avoid re-building layers that don't need to be rebuilt). In practice that means we traded 7 minutes of compute for a 200MB of download. I hope this is an improvement :P